src/zope/exceptions/tests/test_exceptionformatter.py

```diff
@@ -774,6 +778,32 @@ def test_format_exception_as_html(self):
         self.maxDiff = None
         self.assertEqual(expected, result)
 
+    def test_pep302_loader_source_in_traceback(self):
+        sys_path = sys.path[::]
+
+        def restore_sys_path():
+            sys.path[::] = sys_path
+        self.addCleanup(restore_sys_path)
+
+        module_name = self._testMethodName
+        with tempfile.TemporaryDirectory() as td:
+            td = pathlib.Path(td)
+            module_zipfile = td / f"{module_name}.zip"
+            with zipfile.ZipFile(module_zipfile, "w") as zf:
+                zf.writestr(
+                    zipfile.ZipInfo(f"{module_name}/__init__.py"),
+                    """
+def f():
+    1 / 0
+""")
```

Comment:
```suggestion
                    textwrap.dedent("""
                        def f():
                            1 / 0
                    """)
```
This improves readability but additionally requires to import `textwrap`.

Reply:
Thank you, I have applied this and changed another place where `textwrap` was imported locally in the test function to make it a global import.